binrz/rizin/rizin.c

```diff
@@ -4,53 +4,51 @@
 #include <rz_util.h>
 
 static void rz_cmd(int in, int out, const char *cmd) {
-        write (out, cmd, strlen (cmd) + 1);
-        write (out, "\n", 1);
-        int bufsz = (1024 * 64);
-        unsigned char *buf = malloc (bufsz);
-        if (!buf) {
-                return;
-        }
-        while (1) {
-                int n = read (in, buf, bufsz);
-				buf[bufsz - 1] = '\0';
```

Comment:
why was this removed? I think it is important to put a 0 at  the end of the buffer, no?